rflx/generator/generator.py

```diff
@@ -760,14 +760,14 @@ def _integer_functions(self, integer: Integer) -> UnitPart:
         return UnitPart(specification)
 
     def _enumeration_functions(self, enum: Enumeration) -> UnitPart:
-        incomplete = len(enum.literals) < 2**64
+        incomplete = len(enum.literals) < 2**63
 
         specification: ty.List[Declaration] = []
 
         validation_expression = (
             (
                 Less(Variable("Val"), Pow(Number(2), enum.size.ada_expr()))
-                if enum.size.simplified() != expr.Number(64)
+                if enum.size.simplified() != expr.Number(63)
                 else TRUE
             )
```

Comment:
I think it would be good to also have references to #1077 for all uses of 63 in this file. Alternatively, we could also consider adding a constant to `rflx.const` and use this constant at all places in all modules where 63 is now used. It would then be sufficient to only add a reference to this constant definition, and it would also make it clearer where this number comes from.

Reply:
done